packages/jest-cli/src/reporters/__tests__/summary_reporter.test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+const env = Object.assign({}, process.env);
+const write = process.stderr.write;
+const globalConfig = {
+  watch: false,
+};
+const aggregatedResults = {
+  numTotalTestSuites: 1,
+  snapshot: {
+    filesUnmatched: 1,
+    unmatched: 2,
+  },
+  testResults: {},
+};
+
+let results = [];
+let SummaryReporter;
+
+beforeEach(() => {
+  process.env.npm_lifecycle_event = 'test';
+  process.env.npm_lifecycle_script = 'jest';
+  process.stderr.write = result => results.push(result);
+  SummaryReporter = require('../summary_reporter').default;
+});
+
+afterEach(() => {
+  results = [];
+  process.env = env;
+  process.stderr.write = write;
+});
+
+describe('onRunComplete', () => {
```

Comment:
Can we remove the describe and just move its title to the tests? There's too little going on here to benefit from using it